justfile

```diff
@@ -77,23 +77,23 @@ publish-schemas namespace name version: schema
   fi
 
   # check that the metadata exists
-  if [ ! -e "./metadata.json" ]; then \
+  if [ ! -e "./contracts/{{name}}/metadata.json" ]; then \
     echo "Please create metadata.json for module metadata"; exit; \
   fi
 
   tmp_dir="$(mktemp -d)"
-  schema_out_dir="$tmp_dir/{{namespace}}/{{name}}/{{version}}"
-  metadata_out_dir="$tmp_dir/{{namespace}}/{{name}}"
+  schema_out_dir="$tmp_dir/{{namespace}}/{{name}}/"
```

Comment:
This schema out dir includes the namespace while the schema.sh script doesn't seem to do that. Is that the expected behaviour?

Reply:
Why would we need to include namespace in "app_template/schemas"? But we totally should include them when we push to the "AbstractSdk/schemas"